docker/docker-compose.yml

```diff
@@ -120,10 +131,18 @@ services:
       # todo : we are in process of making them as part of default.
       - PINOT_BROKER_TAG=DefaultTenant
       - PINOT_SERVER_TAG=DefaultTenant
-    restart: on-failure
+    volumes:
+      - ../scripts/wait-for-it.sh:/usr/bin/wait-for-it.sh
+    entrypoint: ["sh", "/usr/bin/wait-for-it.sh", "-h", "hypertrace-federated-service", "-p", "9002", "--timeout=120",
```

Comment:
if the shell script is correct, with the right perms (set here and checked into git), you don't need to qualify with "sh" first, just use wait-for-it.sh

Reply:
done.